test/ExecutionEnvironment.t.sol

```diff
@@ -361,6 +330,7 @@ contract ExecutionEnvironmentTest is BaseTest {
         (status,) = address(executionEnvironment).call(postOpsData);
     }
 
+    /*
     function test_solverMetaTryCatch() public {
```

Comment:
Shouldn't this be removed now?

Reply:
Yes but we should also add 2 new tests, for solverPreTryCatch and solverPostTryCatch. I've added a note to refactor this one into those 2 and left it commented as some of this test code may be reused in the new ones. Will add it to the list of tests to do after the audit fixes.